Whitespace in user folder name causes problems with plugin installation

**Node.js version**: 9.8.0

**npm version**: 5.6.0

**Strapi version**: 3.0.0-alpha.11.1

**Operating system**: Windows 10

**Do you want to request a *feature* or report a *bug*?**
Report a bug

**What is the current behavior?**
Currently unable to install plugins on initial build or through the admin marketplace. Believe it has to do with there being whitespace in my users folder. The folder name is Alex Work and it seems to be looking for just Alex.

The code below happens for each plugin:
```

[2018-03-15T23:55:29.305Z] info Installing content-manager...
module.js:545
    throw err;
    ^

Error: Cannot find module 'C:\Users\Alex'
    at Function.Module._resolveFilename (module.js:543:15)
    at Function.Module._load (module.js:470:25)
    at Function.Module.runMain (module.js:690:10)
    at startup (bootstrap_node.js:194:16)
    at bootstrap_node.js:666:3
[2018-03-15T23:55:29.948Z] debug POST /admin/plugins/install (700 ms)
[2018-03-15T23:55:29.957Z] debug GET f5605e1347775a7276c42bee1847195f.svg (6 ms)
```

--- a/packages/strapi-admin/controllers/Admin.js
+++ b/packages/strapi-admin/controllers/Admin.js
@@ -87,7 +87,7 @@ module.exports = {
       strapi.reload.isWatching = false;
 
       strapi.log.info(`Uninstalling ${plugin}...`);
-      exec(`node ${strapiBin} uninstall ${plugin}`);
+      exec(`node "${strapiBin}" uninstall ${plugin}`);
 
       ctx.send({ ok: true });
 

--- a/packages/strapi-admin/controllers/Admin.js
+++ b/packages/strapi-admin/controllers/Admin.js
@@ -54,7 +54,7 @@ module.exports = {
 
       strapi.log.info(`Installing ${plugin}...`);
 
-      exec(`node ${strapiBin} install ${plugin} ${port === '4000' ? '--dev' : ''}`);
+      exec(`node "${strapiBin}" install ${plugin} ${port === '4000' ? '--dev' : ''}`);
 
       ctx.send({ ok: true });
 

